File: 151003/Mytnik/Lab1/Lab1/Lab1/DTO/CreatorRequestTo.cs
using Lab1.Models;
using Lab1.DTO.Interface;

namespace Lab1.DTO
{
    public class CreatorRequestTo(int Id,string? Login, string? Password, string? Firstname, string? Lastname) : IRequestTo
    {
        public int Id { get; set; } = Id;
        public string? Login { get; set; } = Login;

        public string? Password { get; set; } = Password;

        public string? Firstname { get; set; } = Firstname;

        public string? Lastname { get; set; } = Lastname;

      
    }
}

File: 151002/Luschickij/fullProject/DC_lab/publisher/src/main/java/com/luschickij/publisher/dto/user/CreatorRequestTo.java
package com.luschickij.publisher.dto.creator;

import com.luschickij.publisher.model.IdentifiedEntity;
import jakarta.validation.constraints.Size;
import lombok.*;
import lombok.experimental.SuperBuilder;
import org.springframework.beans.factory.annotation.Value;

import java.math.BigInteger;


@EqualsAndHashCode(callSuper = true)
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CreatorRequestTo extends IdentifiedEntity {

    private Long id;
    @Size(min = 2, max = 64)
    String login;
    @Size(min = 8, max = 128)
    String password;
    @Size(min = 2, max = 64)
    String firstname;
    @Size(min = 2, max = 64)
    String lastname;
}
